libnet/configure.ac

```diff
@@ -4,25 +4,50 @@ dnl All rights reserved.
 dnl
 dnl Process this file with autoconf to produce a configure script.
 
-AC_INIT([libnet],[1.2-rc3])
+AC_INIT([libnet],[1.2-rc3],,,[http://github.com/sam-github/libnet])
 AC_MSG_RESULT(beginning autoconfiguration process for libnet-${PACKAGE_VERSION} ...)
 AC_CANONICAL_TARGET
 AC_CONFIG_SRCDIR([src/libnet_build_ip.c])
 AC_CONFIG_MACRO_DIR([m4])
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([foreign subdir-objects no-dependencies -Wall])
 AC_CONFIG_HEADERS([include/config.h])
-AM_MAINTAINER_MODE
 
 dnl
 dnl Check for the usual programs
 dnl
 AC_PROG_CC
 AC_PROG_INSTALL
 AC_USE_SYSTEM_EXTENSIONS
+AX_CFLAGS_WARN_ALL
```

Comment:
Yes it works for me, but you have to have the support file
````
$ dpkg --search ax_cflags_warn_all.m4
autoconf-archive: /usr/share/aclocal/ax_cflags_warn_all.m4
````
You can just download it and slap it in `m4/`, wait I already did that, don't know why `autoreconf` is not finding it for you...